### PR TITLE
Move to helm3

### PR DESCRIPTION
**Pull request by satadruroy**

### CODE REVIEW

File: eks/modules/services/k8s.tf [hunk 1]
```diff
@@ -7,6 +7,7 @@ data "aws_eks_cluster_auth" "eks-auth" {
 }
 
 provider "kubernetes" {
+  version = "~> 1.10.0"
   load_config_file = false
   host = "${data.aws_eks_cluster.eks.endpoint}"
   version                = "1.10.0"
```
Comment (viccuad) [inline eks/modules/services/k8s.tf]:
```suggestion
```

Comment (viccuad) [reply eks/modules/services/k8s.tf]: Terraform fails because version is defined twice

File: gke/k8sresources.tf [hunk 2]
```diff
@@ -1,39 +1,12 @@
 provider "kubernetes" {
+  version = "~> 1.10.0"
   load_config_file = false
   host = "https://${google_container_cluster.gke-cluster.endpoint}"
   version                = "1.10.0"
```
Comment (viccuad) [inline gke/k8sresources.tf]:
```suggestion
```

Comment (viccuad) [reply gke/k8sresources.tf]: Terraform fails because version is defined twice

Comment (satadruroy) [reply gke/k8sresources.tf]: Yes, sorry - I think I didn't resolve the merge conflict correctly. Anyway, have pushed the fixes.

**Pull request merged**